cpio/writerhelper.go

```diff
@@ -148,6 +151,36 @@ func (w *WriterHelper) CopyFileTo(src, dst string) error {
 	return nil
 }
 
+func (w *WriterHelper) TransformFileTo(src, dst string, fn Transformer) error {
+	w.ensureBaseDirectory(path.Dir(dst))
+
+	f, err := os.Open(src)
+	if err != nil {
+		log.Panicf("open failed: %s - %v", src, err)
```

Comment:
remove this - returning the error is fine enough

Reply:
This one is kept for symmetry with the one in `CopyFileTo()`. Would you like me to drop both?

Reply:
You may - I tried to remove all panics across fakemachine in #74 so i can just rebase that on top of your work :-)

Reply:
Added extra commit to drop the pre-existing Panicf